src/cmds/ldpcap/host.go

```diff
@@ -133,8 +133,6 @@ func (h *Host) Write(out io.Writer) {
 	if v := fmtStrings(h.Routers); v != "" {
 		fmt.Fprintf(out, "routers=%v\n", v)
 	}
-
-	fmt.Fprintln(out, "\n")
```

Comment:
This was causing a failure when running tests, saying something along the lines of a redundant newline was being created. If a newline is really needed here, I can replace `Fprintln` with `Fprint`.

Reply:
@jcrussell thoughts on this? Can I leave it out or should I replace it with `Fprint` (or `Fprintln` without the `\n')?

Reply:
Yeah, we should probably add an extra newline somehow. Was it a `go vet` complaint?